### PR TITLE
feat: review static "sign-in" pages

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Review text of the pre-rendered "sign-in" pages with shorterm content.

Add known routes to sitemap.xml.

# Tests

No particular tests regarding static content as it aims to evolve soon.

### CODE REVIEW

File: frontend/src/lib/components/common/SignIn.svelte [hunk 1]
```diff
@@ -49,7 +49,7 @@
 </script>
 
 <button on:click={signIn} data-tid="login-button" class="primary" {disabled}>
-  {$i18n.auth.login}
+  <slot>{$i18n.auth.login}</slot>
```
Comment (lmuntaner) [inline frontend/src/lib/components/common/SignIn.svelte:52]: Why is this needed?

**Review by lmuntaner — APPROVED** [review]
LGTM!

Just a minor question, but already approving.

File: frontend/src/lib/components/common/SignIn.svelte (hunk 1 above)
Comment (peterpeterparker) [reply frontend/src/lib/components/common/SignIn.svelte:52]: It's the correct way to set a default value to a slot in Svelte right? I know it's the default way in pure HTML but now that you ask...

Need it because in case of "Voting" screen the CTA is different. Not "Sign in with II" but "Sign in to vote"

Comment (lmuntaner) [reply frontend/src/lib/components/common/SignIn.svelte:52]: Aaaahh, yes correct.

This is the first time that I see it. Cool!

Comment (lmuntaner) [reply frontend/src/lib/components/common/SignIn.svelte:52]: Yes, it's the correct way: https://svelte.dev/tutorial/slot-fallbacks

Comment (peterpeterparker) [reply frontend/src/lib/components/common/SignIn.svelte:52]: Cool, thanks for double checking!
